code/datums/station_traits/negative_traits.dm

```diff
@@ -410,3 +410,250 @@
 					continue
 
 			CHECK_TICK
+
+///Station traits that influence the space background and apply some unique effects!
+/datum/station_trait/nebula
```

Comment:
This needs `STATION_TRAIT_ABSTRACT` if it's not intended to run, unless the prototype works fine regardless, then whatever